sync/peers.go

```diff
@@ -0,0 +1,26 @@
+package sync
+
+import (
+	"github.com/spacemeshos/go-spacemesh/crypto"
+	"github.com/spacemeshos/go-spacemesh/p2p"
+)
+
+type Peer crypto.PublicKey
+
+type Peers interface {
+	p2p.Service
+	GetPeers() []Peer
+}
+
+type PeersImpl struct {
+	p2p.Service
+	getPeers func() []Peer
```

Comment:
Why not *peer?

Reply:
peer = crypto.PublicKey which is an interface ....